pkg/kp/pcstore/consul_store.go

```diff
@@ -225,22 +225,29 @@ func (s *consulStore) Watch(quit <-chan struct{}) <-chan WatchedPodCluster {
 				}
 			}
 
+			clusters := WatchedPodClusters{
+				Clusters: []*fields.PodCluster{},
```

Comment:
nit: don't think you need to initialize `Clusters`, `append` will do the right thing